pkg/scheduler/api/types.go

```diff
@@ -26,12 +26,6 @@ import (
 )
 
 const (
-	// MaxUint defines the max unsigned int value.
-	MaxUint = ^uint(0)
-	// MaxInt defines the max signed int value.
-	MaxInt = int(MaxUint >> 1)
-	// MaxTotalPriority defines the max total priority value.
-	MaxTotalPriority = int64(math.MaxInt64)
```

Comment:
MaxTotalPriority can stay since it has specific meaning, we may change all of that when we move to Score plugins in the scheduler.

Reply:
I agree with @ahg-g on this, MaxTotalPriority defines a meaning in addition of being equal to MaxInt64 (as it could change or something). Updating it now. 

@Huang-Wei  let me know if you have any further thoughts on this.

Reply:
https://github.com/kubernetes/kubernetes/pull/82732/files#diff-7f99c089b0b9494f4e4dea69979798c2L38 should use MaxTotalPriority instead of MaxInt64

Reply:
correct link: https://github.com/kubernetes/kubernetes/blob/9a7494b96ab84f7a002defe043cedab42a7c0257/pkg/scheduler/api/types.go#L34

Reply:
LGTM.

Reply:
@ahg-g did you mean this line:
https://github.com/kubernetes/kubernetes/blob/92ef49fa8d20a8808c10bd8cda36f975018f84d8/pkg/scheduler/factory/plugins_test.go#L63

Reply:
he meant `MaxWeight` two lines bellow.

Reply:
doesn't it make sense to change the line I referenced as well to `MaxTotalPriority`

Reply:
> doesn't it make sense to change the line I referenced as well to `MaxTotalPriority`

yes it does.